snoop/data/digests.py

```diff
@@ -102,7 +102,12 @@ def index(blob, digests_gather):
     content = _get_document_content(digest)
     version = _get_document_version(digest)
     body = dict(content, _hoover={'version': version})
-    indexing.index(digest.blob.pk, body)
+    try:
+        indexing.index(digest.blob.pk, body)
+    except RuntimeError as e:
+        log.exception(repr(e))
+        log.exception(repr(body))
```

Comment:
`log.exception` already prints the error and stack trace so the 2nd line is enough.